gulpfile.js

```diff
@@ -17,11 +17,53 @@ var appsDir = './apps/';
 var debugDir = './debug/';
 var releaseDir = './release/';
 
-function get_task_name(key) {
-    return 'build-' + key.replace(/([A-Z])/g, function ($1) { return "-" + $1.toLowerCase(); });
+// -----------------
+// Helper functions
+// -----------------
+
+// Get platform from commandline args
+// #
+// # gulp <task> --osx64         to execute task only for macOS platform (--osx64, --win32 or --linux64)
+// # 
+function get_platform_from_args() {
+    var platforms = ['osx64', 'win32', 'linux64'];
+    if (process.argv.length > 3) {
+        var arg = process.argv[3].split('-')[2];
+        if (platforms.indexOf(arg) > -1) {
+            return [ arg ];
+        }
+        else {
+            console.log("Unknown platform: " + arg);
+            process.exit();
+        }
+    }  
+    return platforms;
 }
 
-gulp.task('clean', function () { return del(['./dist/**'], { force: true }); });
+function get_release_filename(platform, ext) {
+    var pkg = require('./package.json');
+    return 'Betaflight-Configurator_' + platform + '_' + pkg.version + '.' + ext;
+}
+
+// -----------------
+// Tasks
+// -----------------
+
+gulp.task('clean', function () { 
```

Comment:
I think the intuitive thing would be to call this one `clean-dist` and have a `clean` task that calls all of the other clean tasks.